### DISCUSSION TITLE
third-party statistics code

### DISCUSSION THREAD

**Discussion by chili89** [YOURLS/YOURLS#3209] [Q&A]
Hello. I'm sorry, my English is very bad, but I need help. I speak Russian.

Tell me where I can insert the code of a third-party metric so that I can collect the data that went to my short address.

Where can I add this code to make it work

<script>
!function(f,b,e,v,n,t,s)
{if(f.fbq)return;n=f.fbq=function(){n.callMethod?
n.callMethod.apply(n,arguments):n.queue.push(arguments)};
if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
n.queue=[];t=b.createElement(e);t.async=!0;
t.src=v;s=b.getElementsByTagName(e)[0];
s.parentNode.insertBefore(t,s)}(window, document,'script',
'https://connect.facebook.net/en_US/fbevents.js');
fbq('init', '11116055555546039');
fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none"
src="https://www.facebook.com/tr?id=1111655555039&ev=PageView&noscript=1"
/></noscript>

Скажите куда я могу вставить код сторонней метрики, что бы я мог собирать данные которые перешли по моему короткому адресу.

**Comment by dgw** [comment]
I really don't think this is possible. You're trying to insert tracking JavaScript into a URL shortener that _ideally_ never even serves the visitor a full page before they're redirected to the intended link.

**Reply by ozh** [reply]
This is possible with a plugin that would redirect ONLY using javascript, using some interstitial page like "You are being redirected to xxxxx" and loading the javascript.

For more info see https://github.com/YOURLS/YOURLS/wiki/Possible-With-A-Plugin

**Reply by dgw** [reply]
@ozh _Technically_ possible, but inadvisable? Ultimately it just seems like a bad idea to try and rely on JavaScript tracking in a PHP app when you could use a stats plugin written in PHP instead.

**Reply by ozh** [reply]
Absolutely :)
